extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -151,6 +151,8 @@ export const StateProperty = {
   PREVIEW_STATE: 'previewState',
   RTL_STATE: 'rtlState',
   SHARE_MENU_STATE: 'shareMenuState',
+  // Shopping State
+  SHOPPING_STATE: 'shoppingState',
```

Comment:
Would `SHOPPING_DATA` better describe what's in this?

Reply:
I agree with you Gabriel, but once again, just following the convention of the other variable names, it seems like the other state data all have the _STATE suffix.